client/components/util/type-util.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * @copyright Copyright 2014 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @fileoverview typeUtil is a javascript library to deal with type conversion.
+ * @author joemu@google.com (Joe Allan Muharsky)
+ */
+
+goog.provide('p3rf.perfkit.explorer.components.util.TypeUtil');
+
+goog.scope(function() {
+const explorer = p3rf.perfkit.explorer;
+
+
+
+/**
+ * Returns true if the provided value can be converted to true.
+ * The boolean value 'true', the string 'true' (in any case), or a 1
+ * are all acceptable.
+ *
+ * @constructor
+ * @export
+ */
+explorer.components.util.TypeUtil = class {
+  constructor() {}
+
+  static getBoolean(value) {
+    if (!goog.isDefAndNotNull(value)) {
+      return null;
```

Comment:
The return type is {boolean|null}? That's a bit unexpected, can you document this? In most typical usage this would be silently treated as "false". 

Or alternatively change it to always return a boolean in the normal path and throw an exception for an invalid input?